controllers/sriovoperatorconfig_controller.go

```diff
@@ -95,16 +95,20 @@ func (r *SriovOperatorConfigReconciler) Reconcile(req ctrl.Request) (ctrl.Result
 		return reconcile.Result{}, nil
 	}
 
-	if enableAdmissionController {
-		// Render and sync webhook objects
-		if err = r.syncWebhookObjs(defaultConfig); err != nil {
-			return reconcile.Result{}, err
-		}
+	// Render and sync webhook objects
+	if err = r.syncWebhookObjs(defaultConfig); err != nil {
+		return reconcile.Result{}, err
+	}
 
-		// Render and sync CA configmap
+	if *defaultConfig.Spec.EnableInjector {
```

Comment:
Add a note that in k8s environment, this should never set to true?

Reply:
In the switchdev PR, a CLUSTER_TYPE environment variable will be added to tell what platform the operator is running on. We can add that condition check after that PR is merged.

Reply:
We already using ENABLE_ADMISSION_CONTROLLER, why add CLUSTER_TYPE in the mix? plus in the future we may enable webhook for vanilla k8s.